src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -150,7 +149,9 @@ protected function load(LoaderInterface $loader, $confDir, $pattern)
     }
 
     /**
-     * @param RouteCollectionBuilder|RoutingConfigurator $routes Is a RouteCollectionBuilder for Symfony <= 4.4
+     * @param RoutingConfigurator $routes
+     * @param $confDir
+     * @param $pattern
```

Comment:
We do not add `@param` which add any value, thats why phpstan is here failing now.

```suggestion
```